e2e/cypress/integration/markdown/markdown_text_spec.js

```diff
@@ -42,16 +37,13 @@ describe('Markdown message', () => {
         };
         cy.apiUpdateConfig(newSettings);
 
-        // # Login as "user-1"
-        cy.apiLogin('user-1');
-
-        // # Start cypress server, and listen for request to get posts
-        cy.server();
-        cy.route('GET', 'api/v4/channels/**/posts*').as('getPosts');
-
-        // # Navigate to app and wait for posts request to finish
-        cy.visit('/');
-        cy.wait('@getPosts', {timeout: TIMEOUTS.HUGE}).should('have.property', 'status', 200);
```

Comment:
I'm not particularly sure why this was added but my guess is that previous tests makes the posts load loner than usual.  So, just do this test with new user.

Reply:
I think that's what I was aiming for initially, but this solves it :)